### DISCUSSION TITLE
Adding bit of humor to linter with funny insults

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible/ansible-lint#2205] [Ideas] [answered]
As soon we will reach the point where the linter will be able to to score the quality of the code, similar to pylint, I think it would be fun to build a list of single lint messages we can print corresponding to the quality of the code.

Feel free to suggest sentences and vote the ones you like, we may pick the more popular ones, unless they are really offensive.

Do add **different** answers for each sentence so we can vote them.

**Comment by ssbarnea** [comment]
`Nobody expects ansible-lint to pass!` - inspired from Monty Python (Spanish Inquisition)

**Comment by ssbarnea** [comment]
It’s just a flesh wound…

**Comment by ssbarnea** [comment]
`If you have linting allergy your life so sad. You run ansible-lint, - it's like russian roulette.` - inspired from Uncle Roger.

**Comment by konstruktoid** [comment]
Classics:
https://www.sudo.ws/repos/sudo/file/tip/plugins/sudoers/ins_goons.h
https://www.sudo.ws/repos/sudo/file/tip/plugins/sudoers/ins_python.h

**Comment by konstruktoid** [comment]
"You shall not pass!" - Gandalf

**Comment by konstruktoid** [comment]
Failing and giving up is always an option.

**Comment by konstruktoid** [comment, marked as answer]
I hope you get paid by the hour...
